main/templates/partials/notifications.html

```diff
@@ -0,0 +1,8 @@
+<div class="notifications site-wide d-none">
```

Comment:
I know this is probably copied over from xpro, but I'm not sure we need this outer element. We only render a single notification (or none at all). We can probably just keep the element inside the `if` tag and move the `site-wide d-none` class names down to that element

Reply:
ok